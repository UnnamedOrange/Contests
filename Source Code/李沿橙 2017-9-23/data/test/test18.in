5
29
7 5 4 6 908 8 839 649 634 757 722 12 744 8 807 3 763 8 891 9 974 6 707 2 814 641 695 770 6 
49
125 966 273 506 349 802 356 4 66 782 766 54 918 766 416 714 83 509 603 766 765 119 444 620 987 784 885 736 79 648 854 328 642 216 15 374 43 842 314 831 458 750 711 841 833 839 695 233 7 
29
974 943 984 935 32 922 36 942 928 10 930 47 971 994 980 46 978 948 913 995 47 955 932 990 961 920 927 966 960 
27
24 546 27 64 28 34 79 21 77 67 626 9 67 58 60 45 62 19 44 17 32 1 63 40 909 79 61 
50
427 874 310 564 603 972 894 592 34 980 284 543 433 311 923 199 154 194 894 894 308 403 926 396 671 277 894 958 296 916 10 436 662 161 920 195 362 125 894 210 74 677 930 438 962 905 918 304 462 877 
