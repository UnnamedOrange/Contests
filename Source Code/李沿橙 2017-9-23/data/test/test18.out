2 3 4 5 6 6 6 7 8 8 8 9 634 649 695 707 722 744 757 763 770 807 814 839 891 908 974 641 12 
4 7 15 43 54 66 79 119 216 273 314 349 356 374 444 458 506 603 642 648 695 711 736 750 765 766 766 766 782 784 831 833 839 841 842 854 918 966 987 885 802 714 620 509 416 328 233 125 83 
10 32 36 46 47 913 922 927 928 930 932 935 942 943 948 955 960 961 966 971 974 978 980 984 990 994 995 920 47 
1 9 17 19 21 24 27 28 32 34 40 44 45 58 60 61 62 63 64 67 67 77 79 79 626 909 546 
10 34 74 125 154 161 194 195 199 277 284 296 304 308 310 311 362 396 427 433 436 438 462 543 564 592 662 671 874 894 894 894 894 894 905 916 918 920 923 926 930 958 962 972 980 877 677 603 403 210 
